IsMauiDeadDead/Pages/Home.razor

```diff
@@ -76,6 +88,15 @@
     private SiteStatus? siteStatus;
     private bool isLoading = true;
 
+    // Define data streams that ismauidead.net might be monitoring
+    private readonly Dictionary<string, string> dataStreamUrls = new()
+    {
+        { "GitHub MAUI Repo", "https://api.github.com/repos/dotnet/maui" },
+        { "NuGet MAUI Package", "https://api.nuget.org/v3-flatcontainer/microsoft.maui.controls/index.json" },
+        { "Microsoft Learn MAUI", "https://docs.microsoft.com/en-us/dotnet/maui/" },
+        { ".NET Release API", "https://api.nuget.org/v3-flatcontainer/microsoft.netcore.app/index.json" }
+    };
```

Comment:
These streams are not what I mean. You need to downlaod the home page of ismauidead.net and check all the urls in the page.

So when requesting the page, read the contenst and collect the urls. Then check each one to make sure they are also working.